tgbot/botmanager/templates/statistics.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'admin/base_site.html' %}
+
+{% block title %}
+User Statistics
+{% endblock %}
+
+{% block content %}
+    <h1>User Statistics</h1>
```

Comment:
Давай вместо User Statistics придумаем какое-то подходящее название на русском , можешь придумать свой вариант или взять этот:
 ```suggestion
    <h1>Статистика количества пользователей по датам</h1>
```